docs/api-docs/channels/channel-webhooks.md

```diff
@@ -1,10 +1,20 @@
 # Channel Webhooks
 
-Developers building third-party sales channels or multi-storefront capabilities might need notification when a channel in BigCommerce changes so that they can perform downstream actions required to set up or deactivate the channel. To facilitate this, we've added `store/channel` webhook events that fire when you create or update a channel. This article assumes that you're familiar with webhooks and introduces channel-related events. For a more general introduction to webhooks, see [Webhooks Overview](/api-docs/getting-started/webhooks/about-webhooks).
+Developers building third-party sales channels or multi-storefront capabilities might need notification when a channel in BigCommerce changes so that they can perform downstream actions required to set up or deactivate the channel. To facilitate this, we've added `store/channel` webhook events that fire when you create or update a channel. This article assumes that you're familiar with webhooks. It introduces channel-related events and is a reference for all BigCommerce channel webhook events and their callback payloads. For an introduction to webhooks on BigCommerce, see [Webhooks Overview](/api-docs/store-management/webhooks/overview#callback-payload). For a more general introduction to webhooks, see [Webhooks Overview](/api-docs/getting-started/webhooks/about-webhooks).
 
 To learn more about sales channels, see the [Channels Overview](/api-docs/channels/guide/overview). To learn more about how sales channels function in the context of multi-storefront or multi-channel sales, see the [channels section of the Multi-Storefront Overview](/api-docs/multi-storefront/overview#channels).
 
-## Channel-related webhook events
+There are three kinds of webhooks events that relate to channels: settings, product assignment, and channel event webhooks.
+
+## Creating a webhook

```

Comment:
@traci707 it would be great if we could add an example of creating scopes for a specific channel and also give an explanation that `store/channel/{channel_id}/cart/created` e.g. here `{channel_id}` is a placeholder to add `channel_id` of channel they would like to subscribe to receive each of the events :)

Reply:
@kristinapototska I added an example, can you take a look?

Reply:
@traci707 @kristinapototska I don't love that we are duplicating information that is presented clearly, efficiently, and in greater detail in the [Webhooks Overview](https://github.com/bigcommerce/dev-docs/pull/1458/files#diff-b939035873497898af2c6147947e37718075bd54917b2e760c542067432a3ea4R13). What is wrong with [the version of this that links instead of duplicating](https://github.com/bigcommerce/dev-docs/pull/1458/files/64d406449c567f03291a7d4aa885a36f305b65de..2a24c69cf7678ec98c179e0c1ea40ef006a3549b#diff-7fb5b3279491fae2946e5efaf437560f0e1b606edc750c852cfe3e4730a12aebL11) along with a callout about replacing placeholders?